### DISCUSSION TITLE
Caption Ketika Mengirimkan Dokumen

### DISCUSSION THREAD

**Discussion by melodiazz** [kirimwa-id/forum#169] [Saran dan Ide]
Hi Pak Rio,

Jika memungkinkan, apakah bisa add caption ketika kirim pesan dalam bentuk dokumen? Saat ini parameter caption ketika message type document berfungsi untuk set nama file kemungkinan karna sebelumnya memang whatsapp juga belum mendukung caption ketika kirim doc. Namun saat ini whatsapp sendiri sudah  mendukung  hal tersebut, maka akan sangat baik jika kirimwa.id juga bisa melakukan hal yang sama.
![image](https://user-images.githubusercontent.com/22655268/222028891-621e694f-2f11-4e05-a384-e4ef8eb4c220.png)

**Comment by rioastamal** [comment]
@melodiazz Terima kasih sarannya, akan dimasukkan roadmap.

**Comment by masbagusstwn** [comment]
Padahal saya sudah lebih dulu bertanya disini https://github.com/kirimwa-id/forum/discussions/162 hihi
